src/Components/Authorization/DesktopModal.tsx

```diff
@@ -7,6 +7,7 @@ import { DesktopHeader } from "./DesktopHeader"
 
 export interface DesktopModalProps extends ModalProps {
   subtitle?: string
+  onTypeChange?: (type: any) => void
```

Comment:
typing this as `any` for now until we decide whether we want to use `register` or `signup`.

Reply:
@xtina-starr let's go with `signup`. I think it's clearer then `register` for the Sign Up Form

Reply:
Was leaning towards signup myself.